test/uk/gov/hmrc/merchandiseinbaggage/stubs/CurrencyConversionStub.scala

```diff
@@ -20,22 +20,22 @@ import java.time.LocalDate.now
 import com.github.tomakehurst.wiremock.WireMockServer
 import com.github.tomakehurst.wiremock.client.WireMock.{aResponse, get, urlMatching}
 import com.github.tomakehurst.wiremock.stubbing.StubMapping
+import com.google.inject.Inject
 import play.api.http.Status.OK
 import play.api.libs.json.Json
 import uk.gov.hmrc.merchandiseinbaggage.CoreTestData
-import uk.gov.hmrc.merchandiseinbaggage.config.CurrencyConversionConfiguration
+import uk.gov.hmrc.merchandiseinbaggage.config.AppConfig
 import uk.gov.hmrc.merchandiseinbaggage.model.api.ConversionRatePeriod
 
-object CurrencyConversionStub extends CurrencyConversionConfiguration with CoreTestData {
+class CurrencyConversionStub @Inject() (config: AppConfig) extends CoreTestData {
```

Comment:
No need to extend `CoreTestData` and inject `AppConfig`. This can be just be `object CurrencyConversionStub {`. Please update, thanks